### DISCUSSION TITLE
Cant find custom meals.

### DISCUSSION THREAD

**Discussion by gramapp** [jlai/fitness-dashboard#18] [General]
Custom meals "at least I think that's what they were called in the web dashboard" are not showing up in the exercisequest. I can see them in the android app under custom tab, and they definitely were there in on the web dashboard. In other words I think exercisequest is showing only custom foods not the custom meals.
Can you please see if custom foods can be implemented?

**Comment by jlai** [comment]
Meals should show up under the "meals" tab of the food page. All meals are custom. You can add/edit meals using the "manage meals" link on the right side of the meals tab.

I think the Android app treats meals as a custom food with a serving size of "1 meal" which seems like a (recent?) workaround for it not supporting meals the way the old website did.

**Reply by gramapp** [reply]
Yes they do! Sorry I did not notice.

**Reply by gramapp** [reply]
also you might be right. I remember a while back I could not find meals in the app and had to use web dashboard to log them. At the time I could not figure out what was going on...
